ftw/noticeboard/tests/test_add_content.py

```diff
@@ -0,0 +1,43 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.noticeboard.tests import FunctionalTestCase
+from ftw.testbrowser import browsing
+from ftw.testbrowser.pages import factoriesmenu
+from ftw.testbrowser.pages import plone
+
+
+class TestContentTypes(FunctionalTestCase):
```

Comment:
Since you add the permission here you might want to add a test asserting that other than Managers and Site Administrators can not add this type. I suggest using https://pypi.org/project/parameterized/ so that the test does not have to be repeated again and again.

Or just go ahead with the PR without changing anything...

Reply:
@busykoala Thanks for the hint. I'll introduce a complete example workflow later, which will map the permission to roles and user. Right now the permissions are there, since they are mandatory for a new content type and it's a good practice to use a specific add permission per type. I'm also gonna introduce lawgiver later, which will map the permissions to actions groubs 😄